tests/__xlsx__/xlsx_demo_data/xl/sharedStrings.xml

```diff
@@ -1234,4 +1234,16 @@
   <si>
     <t>DATEDIF</t>
   </si>
+  <si>
+    <t>this is a very long text to wrap</t>
+  </si>
+  <si>
+    <t>Wrap text:</t>
+  </si>
+  <si>
+    <t>overflow</t>
+  </si>
+  <si>
+    <t>wrap</t>
+  </si>
```

Comment:
I don't think it really matters, but we should also change the `cout` and `uniqueCount` at the top of this file. Should be  `count="466" uniqueCount="415"`